### DISCUSSION TITLE
<Galaxy Brain>

### DISCUSSION THREAD

**Discussion by devpayoub** [community/community#131275] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I answer a lot of questions, and most of my answers were accepted, but I didn't get the achievements.

**Comment by davevad93** [comment, marked as answer]
Hi @devpayoub ,

The `Galaxy Brain` badge is no longer earnable via Community Discussions since a few months ago, the only way to get the achievement now is by answering discussions in other repos on GitHub with open discussions, for more information see [here](https://github.com/orgs/community/discussions/106536).

**Reply by devpayoub** [reply]
they say no longer be available in the Community and they dont say u will get it in others repos

**Reply by davevad93** [reply]
Yes, I was referring to more info about the reasons behind the deactivation of the achievements in the Community, sorry I didn't explain properly! In any case the badge is still earnable outside the Community Discussions. 👍

**Reply by devpayoub** [reply]
ohh ok thnx

**Reply by davevad93** [reply]
You're welcome man!
